.travis.yml

```diff
@@ -31,34 +29,37 @@ script:
 
 jobs:
   include:
-    - php: 5.5
+    # Tests the lowest set of depedencies
+    - php: 7.1
       env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
       install:
         - COMPOSER_MEMORY_LIMIT=-1 travis_retry composer update -n --prefer-lowest --prefer-stable --prefer-dist
 
-    - php: 5.6
-      env: SYMFONY_VERSION="2.8.*"
-
-    - php: 7.0
-      env: SYMFONY_VERSION="3.2.*"
+    # Test against latest Symfony 3.4 stable
+    - php: 7.1
+      env: SYMFONY_VERSION="3.4.*"
 
+    # Test against latest Symfony 3.4 dev version
```

Comment:
Will there ever be a 3.4-RC version again? Should it be `stability=dev` below?

Reply:
I've updated it to `dev` below.